deployment/local/hpa.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: statistics-dashboard-ui-hpa
+spec:
+  maxReplicas: MAX_REPLICAS
+  minReplicas: MIN_REPLICAS
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: statistics-dashboard-ui-deployment
+  behavior:
```

Comment:
I would say you could leave the behavior for the scaling to the default value e.g. not supplying anything here for that section.  
Unless there is a specific reason for it at this stage?  
Same goes for all environments.